### ISSUE TITLE
Multiple cursors does not work

### ISSUE THREAD

**Issue by Mandras** [emmetio/sublime-text-plugin#122]
If you have multiple cursors, it does not work properly.
It will take only the first one
Previous version was working fine with it

**Comment by Mandras** [comment]
Additionnal information: It doesn't work with 1 letter HTML tag, like a, u, i, p, ...

**Comment by Vanderhoof** [comment]
This is very frustrating! I'm so used to it, please return multiple cursors support!

**Comment by sergeche** [comment]
Fixed in v2.2.0, please check if it works as expected

**Comment by miltonmclai** [comment]
Thanks, it works better now.  Both "tab to insert" and "Wrap with Abbreviation" can work with multiple cursors now.  However, 1 more minor fixes will make it perfect.

In previous version, after using "Wrap with Abbreviation" with multiple cursors, the multiple cursors stay right after the inner most opening tag.

But now only the first cursor left and jump to right after the end of outer most closing tag.  This behaviour makes us not able to continue the multiple cursors points editing after using "wrap".

See if it is possible to fix.  Thanks.

**Comment by sergeche** [comment]
Fixed multiple cursors in v2.2.4. Please check if it works as expected

**Comment by miltonmclai** [comment]
Wow, very nice.  We now have the multiple cursors remain after the "wrap" action, this dramatically improve the workflow.  Thank you very much.

May be my description is not good enough.  Let me rephrase them with examples:

In previous version after applying the "wrap", the multiple cursors will stay right inside at the beginning of the inner most opening tags:
<p class="wrapping-tag">(cursor here)wrapped content</p>
<p class="wrapping-tag">(cursor here)wrapped content</p>

In the current version, after the fixing in v2.2.4, the multiple cursors jump to right after the end of outer most closing tags:
<p class="wrapping-tag">wrapped content</p>(cursor here)
<p class="wrapping-tag">wrapped content</p>(cursor here)

I personally like the behaviour of previous version because it allows to do adjustment quicker to the content (wrapped content), for example:
1) closer to the opening tag of the wrapped content that allows quicker fine tune the tag attributes
2) quicker to expand selection to the wrapped content and wrapping tags

Nice to see the further adjustment if you find my comments make sense to you too.

**Comment by sergeche** [comment]
Can you give a complete example of expected behaviour with code snippets? E.g. “when I wrap `foo` text with `ul>li` abbreviation I expect caret to be placed at…

```html
<ul>
  <li>foo</li>|
</ul>
```

**Comment by miltonmclai** [comment]
Let me try (first time using code block...).  In the previous version, when I wrap `foo` text with `ul>li` abbreviation, caret was placed at:

```
<ul>
   <li>|foo</li>
</ul>
```

If I wrapped 2 `foo`s in a sentence "Hi foo, my name is also foo." with `span`, it should be:
```
Hi <span>|foo</span>, my name is also <span>|foo</span>.
```
